src/GLM.jl

```diff
@@ -1,12 +1,13 @@
 __precompile__()
 
 module GLM
+    using ArgCheck
```

Comment:
OK -- I think it might be good to move to ArgCheck for the whole package, but that's definitely outside the scope of what I'm doing here